tests/unit/plugins/cache/test_redis.py

```diff
@@ -25,8 +25,21 @@
 
 from ansible.plugins.loader import cache_loader
 from ansible_collections.community.general.plugins.cache.redis import CacheModule as RedisCache
+from ansible import constants as C
+from ansible.release import __version__ as ansible_version
 
 
 def test_redis_cachemodule():
     # The _uri option is required for the redis plugin
-    assert isinstance(cache_loader.get('community.general.redis', **{'_uri': '127.0.0.1:6379:1'}), RedisCache)
+    connection = 'ni 127.0.0.1:6379:1'
+    if ansible_version.startswith('2.9.'):
+        C.CACHE_PLUGIN_CONNECTION = "c." + connection
```

Comment:
```suggestion
        C.CACHE_PLUGIN_CONNECTION = connection
```
What's the `c.` doing there?

Reply:
(Same question for the `ni ` above.)

Reply:
I had a test case in the company's code with Rammstein's lyrics once. :-)

Reply:
Removed